### DISCUSSION TITLE
Run watch command with --keep-intermediates

### DISCUSSION THREAD

**Discussion by HiramTheHero** [tectonic-typesetting/tectonic#1080] [Q&A] [answered]
Hello all. 

Thank you for the great project.

How do I run `tectonic -X watch` with the `--keep-intermediates` option from the `build` command? I couldn't find any documentation that clarified how the `tectonic -X watch --exec` function should be formatted. I apologize if I've missed the information somewhere.

Thank you!

**Comment by pkgw** [comment, marked as answer]
I believe it would be:

```
tectonic -X watch --exec "build --keep-intermediates"
```

I think? The documentation could definitely use some examples to help people out, here!

**Reply by HiramTheHero** [reply]
Works like a charm. Thank you. I was using version 0.13.0 and was having issues. Version 0.14.1 works great.

Thank you again for the help.
